equinoxfitness/mssql-runner
# Datplat 1712 add default from date

## Description
get default from date from a global variable or system environment variable

## Changes

- Up version to 0.1.2
- Add default from date in init.py file

### How Has This Been Tested?


_Example:_
1. Checkout this branch

1. Switch to virtual environment

1. Run `python -m mssql_runner.module -s "sample/mssql_runner_test.sql" -p "var1-cat, var2-dog" -b '9999'`

- [ ] Verify that sample_table is populated with latest data
